Example/Source/Examples/Interactive/InsertBrickViewController.swift

```diff
@@ -49,47 +67,38 @@ class InsertBrickViewController: BrickApp.BaseBrickController, HasTitle {
         self.navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(InsertBrickViewController.insertBrick))
 
         let section = BrickSection(Section, bricks: [
-            SegmentHeaderBrick(dataSource: self, delegate: self),
-            LabelBrick(BrickIdentifiers.repeatLabel, backgroundColor: UIColor.lightGray, dataSource: self)
+            BrickSection(bricks: [SegmentHeaderBrick("AppearSegmentHeaderBrick", dataSource: self, delegate: self),
```

Comment:
Can you make the string into constants?
Can you also align the bricks, so it's a bit more readable?
```
        let section = BrickSection(Section, bricks: [
            BrickSection(bricks: [
                        SegmentHeaderBrick("AppearSegmentHeaderBrick", dataSource: self, delegate: self),
                        SegmentHeaderBrick("InsertSegmentHeaderBrick", dataSource: self, delegate: self)
            ]),
            LabelBrick(BrickIdentifiers.repeatLabel, height: .fixed(size: 37), backgroundColor: UIColor.lightGray, dataSource: self)
            ], inset: 10, edgeInsets: UIEdgeInsets(top: 5, left: 5, bottom: 5, right: 5))
```